lib/adapter.js

```diff
@@ -153,6 +153,13 @@ function doNextCompaction(self) {
   });
 }
 
+function attachmentNameError(name) {
+  if (name.charAt(0) === '_') {
+    return 'Attachment name can\'t start with \'_\'';
```

Comment:
Why not to include bad attachment name into error? This will save time to figure out what exactly caused the issue.